images/hub/jupyterhub_config.py

```diff
@@ -129,6 +129,10 @@
 if init_containers:
     c.KubeSpawner.init_containers.extend(init_containers)
 
+extra_containers = get_config('singleuser.extra-containers')
+if extra_containers:
+    c.KubeSpawner.extra_containers.extend(extra_containers)
+
```

Comment:
Thanks for positioning this code addition in a very logically consistent place within the config =D just below init-containers like in other places.